deployment/k8s/modules/container.pkl

```diff
@@ -49,7 +50,7 @@ container: PodSpec.Container = new {
         }
         new {
             name = "NEXT_PUBLIC_ORYCLOUD"
-            value = read("env:ORYCLOUD")
+
```

Comment:
issue(blocking): please, avoid changing the deployment files, this would have consequences in the deployment and would break the environments.